CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 3.0.0 (2021-03-??)
+
+- feat: merge `next-on-netlify` into `@netlify/plugin-nextjs`
+  
+  This improves our ability to maintain the plugin, reduces confusion about how to configure Next.js support for Netlify builds, and prepares the plugin for something we're pretty dang excited about: automatic installation for new Next.js projects!
+
+> **NOTE: There are no breaking changes for existing plugin users.** This is a major release because we're bringing an external package into the plugin _and_ turning on automatic installation for new Next.js projects on Netlify. This doesn't break anything, but it's a big enough change that we wanted to treat it as a major release.
+
```

Comment:
note to (possibly?) include a migration from NoN somewhere in this PR (likely after it's added to the NoN repo itself)

Reply:
@lindsaylevine that's a good idea. maybe it could be a link to the NoN repo's migration guide so we keep a single source of truth?